subiquity/server/controllers/locale.py

```diff
@@ -37,7 +37,7 @@ def load_autoinstall_data(self, data):
     def start(self):
         self.model.selected_language = os.environ.get("LANG") \
                 or self.autoinstall_default
-        self.configured()
+        self.app.aio_loop.create_task(self.configured())
```

Comment:
Application.start is already async, it would be in future work to see if the controller.start() methods could be async also.
I guess this is more of an asyncio question, but is there any risk that multiple tasks are trying to modify the _configured_names set?

Reply:
My mental model for the controller start() methods is that it is when to start doing async things, but shouldn't really be async themselves -- we don't want to block here. But maybe that's not helpful.

SubiquityModel._configured is not async (or, more precisely, doesn't await on anything) so it's always executed serially.